flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/co/CoProcessFunction.java

```diff
@@ -50,7 +46,7 @@
  * @param <OUT> Output type.
  */
 @PublicEvolving
-public interface TimelyCoFlatMapFunction<IN1, IN2, OUT> extends Function, Serializable {
+public interface CoProcessFunction<IN1, IN2, OUT> extends Function, Serializable {
 
 	/**
 	 * This method is called for each element in the first of the connected streams.
```

Comment:
<p>This function can output zero or more elements using the {@link Collector} parameter
	 * and also update internal state or set timers using the {@link Context} parameter.